_projects/decrease_docker_image_size.md

```diff
@@ -0,0 +1,76 @@
+---
+collaborating_projects:
+  - docker-coala-base
+desc: "Find creative ways to decrease the coala Docker image size"
+issues:
+  - https://github.com/coala/docker-coala-base/issues/55
+difficulty: "hard"
+initiatives:
+  - GCI
+markdown: decrease_docker_image_size.md
+mentors:
+  - yukiisbored
+name: "Decrease Docker image size"
+tags:
+  - coala
+  - docker
+  - linux
+---
+
+## Task
+
+Submit a Pull Request to the [docker-coala-base][repo] project which decreases
+the image size.
+
+## Objective
+
+This task is considerably hard which requires creativity and knowledge for
+Docker and Linux to decrease the Docker Image size.
+
+## Background
+
+The Docker image for coala is huge (around ~2G!) which makes it a huge drawback
+to be used especially for those who are with limited Internet access.
+
+Decreasing it isn't easy because a lot of linters requires a lot of dependencies
```

Comment:
Phrasing correction:
-> `Decreasing it isn't easy because a lot of linters (and some edge cases) requires a lot of dependencies`
delete next line